_events/sr2019/competition.md

```diff
@@ -3,24 +3,46 @@ title: Competition
 date: 2019-04-06 9:00:00
 layout: event
 type: competition
-location: Southampton University Highfield Campus
+location: Students' Union Building<br>Southampton University Highfield Campus
 ---
 
+## Summary
+
 The Student Robotics Competition marks the ultimate stage of SR2019, showcasing the months of preparation. Our teams will work hard to push their robots to the limit and quickly make repairs between league matches. We will finish the weekend with the finals where the best bots will be named victorious.
 
-## Directions
+## Schedule
```

Comment:
Assuming you mean the finished schedule. I'm planning to use a table as in previous years https://studentrobotics.org/events/sr2017/competition/

Reply:
I was thinking more about the fact that such a table would be quite tall and wondering how it would interact with the other elements on the page. However let's defer worrying about that until we have the information needed for the table.